pyproject.toml

```diff
@@ -11,11 +11,14 @@ python-dateutil = "^2.8.1"
 urllib3 = "^1.26.2"
 aiohttp = "^3.7.3"
 pyyaml = "^5.3.1"
```

Comment:
has pyjwt been considered? (it seems more popular on GH)

Reply:
jose has been used in the legacy and the Elias's POC, so I chose it subjectively. I will have a look at pyjwt and see if we should use it instead.